docs/APISocketOptions.md

```diff
@@ -667,17 +669,18 @@ for more details.
 | --------------------- | ----- | ------- | ---------- | ------ | -------- | ------ | --- | ------ |
 | `SRTO_LATENCY`        | 1.0.2 | pre     | `int32_t`  | ms     | 120 *    | 0..    | RW  | GSD    |
 
-- This flag sets both `SRTO_RCVLATENCY` and `SRTO_PEERLATENCY` to the same value. 
-Note that prior to version 1.3.0 this is the only flag to set the latency. However 
-this is effectively equivalent to setting `SRTO_PEERLATENCY`, when the side is 
-sender (see `SRTO_SENDER`), and `SRTO_RCVLATENCY` when the side is receiver. 
-Bidirectional stream sending in version 1.2.0 was not supported.
+- This option sets both [`SRTO_RCVLATENCY`](#SRTO_RCVLATENCY) and [`SRTO_PEERLATENCY`](#SRTO_PEERLATENCY)
+to the same value specified.
 
+- Prior to SRT version 1.3.0 `SRTO_LATENCY` was the only option to set the latency.
+However it is effectively equivalent to setting `SRTO_PEERLATENCY` on the sender 
+(see [`SRTO_SENDER`](#SRTO_SENDER)), and `SRTO_RCVLATENCY` on the receiver.
+SRT version 1.3.0 and higher support bidirectional transmission, so that each side can 
+be sender and receiver at the same time, and `SRTO_SENDER` became redundant.
```

Comment:
That's not the main reason to have two different options PEERLATENCY/RCVLATENCY. The reason is that since 1.3.0 and bidirectional support, the latency setting is **per direction** - that exactly thing should be mentioned.

Reply:
Changed to
> However it is effectively equivalent to setting `SRTO_PEERLATENCY` in the sending direction
(see [`SRTO_SENDER`](#SRTO_SENDER)), and `SRTO_RCVLATENCY` in the receiving direction.